src/components/admin/ErrorList.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import {List, ListItem, ListItemText} from "@mui/material";
+
+interface RowData {
+    email: string;
+    courseCompletion: number;
+    courseEngagement: number;
+    projectDegree: number;
+    teamProjectDegree: number;
+    bonusProjectUrls: string;
+}
+
+export interface ErrorData {
+    row: RowData;
+    errors: {
+        [key: string]: {
+            row: number;
+            field: any;
+            error: boolean;
+            message: string;
+        }[];
+    };
+}
+
+const errors: ErrorData[] = [
+    {
+        row: {
+            email: "milena@gmail.com",
+            courseCompletion: 5,
+            courseEngagement: 5,
+            projectDegree: 5,
+            teamProjectDegree: 5,
+            bonusProjectUrls:
+                "https:/example.com/link1;              https://example.com/link2; https://example.com/link3",
```

Comment:
tutaj eslint/prettier nie zadziałały z enterami i tymi spacjami czy musi tak być, żeby się potem wyświetlało na www poprawnie?

Reply:
to jest tylko przykładowy obiekt do testów, by sprawdzić czy wyświetlanie listy będzie działało jak FE otrzyma z BE obiekt błędów, docelowy będzie przysyłany z BE w res, wtedy ten usuniemy